sale_double_validation/models/company.py

```diff
@@ -12,13 +12,13 @@ class Company(models.Model):
             ('one_step', 'Confirm sale orders in one step'),
             ('two_step', 'Get 2 levels of approvals to confirm a sale order')
         ],
-        string="Levels of Approvals",
+        string="Levels of Sale Order Approvals",
```

Comment:
I guess the changes in this file are irrelevant.

Reply:
I put here the commit from #1278 to pass tests failing for these errors in main branch:
```
2020-10-05 16:21:41,823 181 WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (so_double_validation, po_double_validation) of res.company() have the same label: Levels of Approvals.
2020-10-05 16:21:41,823 181 WARNING openerp_test odoo.addons.base.models.ir_model: Two fields (so_double_validation_amount, po_double_validation_amount) of res.company() have the same label: Double validation amount.
```
When merged #1278, I'll rebase.

Reply:
@yostashiro done